src/ansys/fluent/core/session_solver.py

```diff
@@ -299,7 +299,6 @@ def _sync_from_future(self, fut: Future):
                 fut_session = fut.result()
             except Exception as ex:
                 raise RuntimeError("Unable to read mesh") from ex
-            state = self.settings.get_state()
```

Comment:
On a different topic: In `read_case_lightweight()` would it make more sense to start the future before the local read? @hpohekar @mkundu1 @prmukherj